utils/merge_syft_sbom.py

```diff
@@ -134,8 +134,8 @@ def merge_sboms(cachi2_sbom_path: str, syft_sbom_path: str) -> str:
 
     syft_sbom["components"] = filtered_syft_components + cachi2_sbom["components"]
 
-    syft_sbom.get("metadata", {}).get("tools", []).extend(
-        cachi2_sbom.get("metadata", {}).get("tools", [])
+    syft_sbom["metadata"]["tools"]["components"].extend(
+        cachi2_sbom["metadata"]["tools"]["components"]
```

Comment:
Shouldn't the script be able to handle any CycloneDX version? Besides, the 1.4 style is still valid in 1.5: https://cyclonedx.org/docs/1.5/json/#metadata_tools

Reply:
That's a good point. I still think Cachi2 output the 1.5 tools format since 1.4 is valid, but already deprecated. 

I can still improve the script to handle both versions if you think it's worth the effort (it will take more effort to test it).

Reply:
A simple unit test would probably be enough no?

Reply:
Depending on how  you release the changes, you may have to bump the versions of the prefetch-dependencies task and every task which uses the merge script.

Outputting v1.5 tools metadata breaks previous versions of the buildah task (and others that use the merge script). Changing the merge script to support only v1.5 breaks previous versions of the prefetch task.

But if you release both at the same time, maybe nobody will notice and you can get away with not versioning the breaking changes